SharpCompress/Writer/Zip/ZipCompressionInfo.cs

```diff
@@ -0,0 +1,47 @@
+﻿using SharpCompress.Common;
+using SharpCompress.Common.Zip;
+using SharpCompress.Compressor.Deflate;
+
+namespace SharpCompress.Writer.Zip
+{
+    public class ZipCompressionInfo
```

Comment:
If this class is intended for internal usage only from the `ZipWriter` class, why not declare it `internal` instead of making the properties `internal`. Alternatively, you can make it a `private` inner class of the `ZipWriter` class for better encapsulation. Call me nit-picking, but I would like this pull request approved in the official repo as well. :smile: